internal/supervisor/installation.go

```diff
@@ -1361,6 +1361,12 @@ func (s *InstallationSupervisor) finalCreationTasks(installation *model.Installa
 	return model.InstallationStateStable
 }
 
+func (s *InstallationSupervisor) finalUpdateTasks(installation *model.Installation, logger log.FieldLogger) string {
```

Comment:
The Installation is updated several times during the creation.
It starts in state `InstallationStateCreationRequestd` then it goes to `InstallationStateCreationPreProvisioning`, `InstallationStateCreationInProgress` etc.

Not sure if that was your intention but we would basically need to have a metric for each state transition.

Reply:
My intention was to be able to track every update action and setup the base for it. I totally agree that we need to track every state transition as we can find bottlenecks.